python/baseline/services.py

```diff
@@ -173,18 +178,19 @@ def task_name(cls):
     def signature_name(cls):
         return 'predict_text'
 
-    def predict(self, tokens):
+    def predict(self, tokens, preproc=False):
         """Take tokens and apply the internal vocab and vectorizers.  The tokens should be either a batch of text
         single utterance of type ``list``
         """
         token_seq, mxlen, mxwlen = self.batch_input(tokens)
         self.set_vectorizer_lens(mxlen, mxwlen)
         examples = self.vectorize(token_seq)
-
+        if preproc:
+            examples['tokens'] = [" ".join(x) for x in tokens]
         outcomes_list = self.model.predict(examples)
         results = []
         for outcomes in outcomes_list:
-            results += [list(map(lambda x: (x[0], x[1].item()), sorted(outcomes, key=lambda tup: tup[1], reverse=True)))]
+            results += [list(map(lambda x: (x[0], x[1]), sorted(outcomes, key=lambda tup: tup[1], reverse=True)))]
```

Comment:
was there a reason that we dont need `item()` here anymore?

Reply:
I had a discussion with @blester125 about this. seems the pyt local models here returns a tensor whereas the remote returns a scalar. so a better idea would be to make the remote models behave the same as the local.